docs/DeveloperGuide.md

```diff
@@ -371,49 +366,135 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 * 2a. The given index is invalid.
     * 2a1. Rapportbook shows an error message.
       Use case resumes at step 2.
-
 * 2b. No fields were specified
     * 2b1. Rapportbook shows an error message.
       Use case resumes at step 2.
-
 * 2c. Updated values is not in the right format.
     * 2c1. Rapportbook shows an error message.
       Use case resumes at step 2.
+* 2d. No index was specified.
+    * 2d-a. Rapportbook has a [contact shown](#use-case-show-contact ).
+      * Use case resumes at step 3.
+    * 2d-b. Rapportbook does not have a [contact shown](#use-case-show-contact ).
+      * 2d-b1. Rapportbook shows an error message.
+      * Use case resumes at step 2.
 
 #### Use case: Filter contacts
-1. User requests to filter contacts of a certain tag and/or name
-2. Rapportbook shows a list of contacts that matches his filter query.
- Use case ends.
+1. User requests to filter contacts of a certain tag and/or name.
+2. Rapportbook shows a list of contacts that matches his filter query.  
+Use case ends.
 
 **Extensions**
 
-* 1a. The tag specified does not exist
-	* 1a1. Rapportbook shows an error message
+* 1a. The tag specified does not exist.
+	* 1a1. Rapportbook shows an error message.
 	* Use case resumes at step 1.
-* 1b. The name specified does not exist
-	* 1a1. Rapportbook shows an error message
+* 1b. The name specified does not exist.
+	* 1a1. Rapportbook shows an error message.
 	* Use case resumes at step 1.
 
 	Use case ends.
 
-
 #### Use case: Clearing filters
 1. User requests to clear a filter that was applied.
-2. Rapportbook shows a list of contacts that without the filter applied.
- Use case ends.
+2. Rapportbook shows a list of contacts that without the filter applied.  
+Use case ends.
 
 **Extensions**
 
-* 1a. The tag specified does not exist
-	* 1a1. Rapportbook shows an error message
+* 1a. The tag specified does not exist.
+	* 1a1. Rapportbook shows an error message.
 	* Use case resumes at step 1.
-* 1b. The name specified does not exist
-	* 1a1. Rapportbook shows an error message
+* 1b. The name specified does not exist.
+	* 1a1. Rapportbook shows an error message.
 	* Use case resumes at step 1.
 
 	Use case ends.
 
-*{More to be added}*
+#### Use case: Show contact
+1.  User [lists contacts](#use-case-list-contacts ).
+2.  User requests to show a specific contact in the list.
+3.  Rapportbook displays the contact in a separate panel for the user.  
+Use case ends. 
+
+**Extensions**
+
+* 1a. The list is empty.
+  Use case ends.
+
+* 2a. The given index is invalid.
+    * 2a1. Rapportbook shows an error message.
+    * Use case resumes at step 2.
+
+#### Use case: Create tag(s)
+1. User requests to create a certain tag or multiple tags.
+2. Rapportbook creates the corresponding tag(s).
+3. Rapportbook shows success message.
+   Use case ends.
+
+**Extensions**
+
+* 2a. At least one of the specified tag(s) already exist.
+    * 2a1. Rapportbook shows a message indicating that the tag(s) already exists.
```

Comment:
Not consistent with the existing sub-points seen in [lines 376-378](https://github.com/AY2223S1-CS2103T-T13-2/tp/blob/afc193a2eaa2349b86f8080caf94176a43465e5e/docs/DeveloperGuide.md?plain=1#L376-L378). Also I feel like `2a-i.` would be better than `2a1.`.

Reply:
I am not sure about the convention that we should follow regarding this. In the case of lines 376-378, they are a further sub-section rather than a sequence of steps, hence the inconsistency.